priv/repo/migrations/20170504190323_create_tasks.exs

```diff
@@ -0,0 +1,24 @@
+defmodule CoursePlanner.Repo.Migrations.CreateTasks do
+  use Ecto.Migration
+
+  def change do
+    execute("""
+          CREATE TYPE task_status AS ENUM (
+            'Pending',
+            'Accomplished'
+          )
+    """)
+    create table(:tasks) do
+      add :name, :string
+      add :deadline, :date
```

Comment:
so far what we see as a task is something that should be done on a date and a period of time. is not important to give a deadline. but rather start and finish. think about it like this "i am available from 1999.1.1 10am till 11am to print course materials